ci/create-aro-cluster.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+echo "Installing Azure CLI"
+
+curl -L https://aka.ms/InstallAzureCli | bash
+
+echo "Installing oc binary"
+
+curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar zxvf - oc
+
+chmod +x oc
+
+echo "Setting config variables"
+
+CLUSTER_NAME="osde2e-$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 6 | head -n 1)"
+
+echo $CLUSTER_NAME > ${SHARED_DIR}/cluster-name
+
+LOCATION=$2
+PULL_SECRET_FILE="/usr/local/osde2e-credentials/stage-ocm-pull-secret"
+ 
+if [ "$LOCATION" == "" ];
+then
+    LOCATION="eastus"
+fi
+
+# just automating the steps in https://docs.microsoft.com/en-us/azure/openshift/tutorial-create-cluster
+
+echo "Logging into Azure"
+
+az login --service-principal --username "$(cat /usr/local/osde2e-credentials/aro-app-id)" --password "$(cat /usr/local/osde2e-credentials/aro-password)" --tenant "$(cat /usr/local/osde2e-credentials/aro-tenant)"
```

Comment:
should this better be set as an environment variable?

Reply:
I'm not trying to say what's right or wrong, but there are security benefits to reading it from a file, most notably that the env isn't inherited by subprocesses that don't need the creds and that the creds aren't visible in `/proc`.

Reply:
The way Prow mounts secrets into a container is as a file, so there's not really a choice here :)